### ISSUE TITLE
EnableSdkContainerDebugging causes build failure if ContainerImageTags is set

### ISSUE THREAD

**Issue by Cjewett** [dotnet/sdk-container-builds#564]
When setting EnableSdkContainerDebugging to true it breaks our compilation because we have ContainerImageTags set. The error is `Both ContainerImageTag and ContainerImageTags were provided, but only one or the other is allowed`. If I remove EnableSdkContainerDebugging the error goes away, but then the launchProfile for SdkContainer no longer works.

```
<PropertyGroup>
  <EnableSdkContainerSupport>true</EnableSdkContainerSupport>
  <EnableSdkContainerImage>true</EnableSdkContainerImage>
  <ContainerBaseImage>mcr.microsoft.com/dotnet/aspnet:8.0.0-jammy</ContainerBaseImage>
  <ContainerImageTags>latest;$(Version)</ContainerImageTags>
</PropertyGroup>
```

**Comment by baronfel** [comment]
@Cjewett this is with the VS/VSCode tooling I expect? If so then `ContainerImageTags` may need to have a Condition added if/when the container debugging is enabled :-/

**Comment by Cjewett** [comment]
@baronfel Yeah, this is with Visual Studio. I'll try to circle back this weekend to see if wrapping things with conditions will offer a temp solution.

**Comment by pastrasigns** [comment]
Is this still an issue?
